lib/interfaces/bdd.js

```diff
@@ -99,6 +113,12 @@ module.exports = function(suite){
       mocha.grep(suite.fullTitle());
       return suite;
     };
+    context.describe.only.noRandom =
```

Comment:
FWIW in one of my PRs that I forgot to merge, I refactored this stuff into a "base" interface, so we don't repeat this code everywhere